### ISSUE TITLE
Auto Update/Refresh Trainer preview window

### ISSUE THREAD

**Issue by ZeroCool940711** [faceshiftlabs/DeepFaceLab#114]
Would be nice if instead of having to press `P` to refresh/update the preview window when training we could just set it to refresh automatically every few seconds or after `X` number of iterations, that way we can keep the preview window on another monitor and see how the progress of the training goes without having to manually press a key to see the changes, some times I forget I have to press the `P` key to update the preview window and I find myself thinking the training haven't done anything new and its stuck just to remember a few seconds or sometimes minutes later that I have to press the `P` key to update it.

**Comment by jmhummel** [comment]
Yes, I agree the training preview needs an overhaul, and in addition to automatic refresh, I'd like to add the ability to go back, and view previously display previews as well, so that you can go back and look at previews from earlier in training.
